src/components/selectionBoxLayer.ts

```diff
@@ -10,29 +11,22 @@ export module Components {
       topLeft: { x: 0, y: 0 },
       bottomRight: { x: 0, y: 0 }
     };
-    private _boxLeftDataValue: number | { valueOf(): number };
-    private _boxRightDataValue: number | { valueOf(): number };
-    private _boxTopDataValue: number | { valueOf(): number };
-    private _boxBottomDataValue: number | { valueOf(): number };
+    private _xExtent: (number | { valueOf(): number })[];
+    private _yExtent: (number | { valueOf(): number })[];
     private _xScale: QuantitativeScale<number | { valueOf(): number }>;
     private _yScale: QuantitativeScale<number | { valueOf(): number }>;
     private _adjustBoundsCallback: ScaleCallback<QuantitativeScale<number | { valueOf(): number }>>;
+    protected _xBoundsMode = PropertyMode.PIXEL;
+    protected _yBoundsMode = PropertyMode.PIXEL;
 
     constructor() {
       super();
       this.addClass("selection-box-layer");
       this._adjustBoundsCallback = () => {
-        this.bounds({
-          topLeft: {
-            x: this._xScale ? this._xScale.scale(this._boxLeftDataValue) : this._boxBounds.topLeft.x,
-            y: this._yScale ? this._yScale.scale(this._boxTopDataValue) : this._boxBounds.topLeft.y
-          },
-          bottomRight: {
-            x: this._xScale ? this._xScale.scale(this._boxRightDataValue) : this._boxBounds.bottomRight.x,
-            y: this._yScale ? this._yScale.scale(this._boxBottomDataValue) : this._boxBounds.bottomRight.y
-          }
-        });
+        this.render();
       };
+      this._xExtent = [undefined, undefined];
+      this._yExtent = [undefined, undefined];
```

Comment:
Previous behavior had the extent return an undefined array, though.  This would introduce a change in behavior